pkg/generate/code/set_resource.go

```diff
@@ -340,7 +347,9 @@ func ListMemberNameInReadManyOutput(
 //      }
 //      if elem.AtRestEncryptionEnabled != nil {
 //          ko.Status.AtRestEncryptionEnabled = elem.AtRestEncryptionEnabled
-//      }
+//      } else {
+//			ko.Status.AtRestEncryptionEnabled = nil
+//		}
```

Comment:
nit, you can fix in a later PR... the indentation above is off, probably because gofmt doesn't replace TAB characters in comments with spaces and the other comments use 4 space indents...

Reply:
nice catch, easy to forget, fixed it